apps/web/src/pages/activities/components/ActivityGraph.tsx

```diff
@@ -0,0 +1,349 @@
+
+        updateTableInnerHtml(tooltipEl, innerHtml);
+      }
+
+      updateToolTipStyles(context, tooltipEl, tooltipModel);
+    },
+  };
+}
+
+function createGradientColor() {
+  return (context: ScriptableContext<'bar'>) => {
+    const { ctx } = context.chart;
+    const gradient = ctx.createLinearGradient(0, 0, 0, 200);
+
+    gradient.addColorStop(0, '#DD2476');
+    gradient.addColorStop(1, '#FF512F');
+
+    return gradient;
+  };
+}
+
+function hideEverySecondTickLabel(this: any, index, val) {
+  return index % 2 === 0 ? this.getLabelForValue(val) : '';
+}
+
+function buildChartDateLabels(data: IActivityGraphStats[]): string[][] {
+  return data.map((x) => {
+    const titleDate = moment(x._id);
+
+    return [titleDate.format('ddd'), `${titleDate.date()}/${titleDate.month() + 1}`];
+  });
+}
+
+function buildChartData(data: IActivityGraphStats[]) {
+  return data.map((x) => {
+    return x.count;
+  });
+}
+
+const StyledBar = styled(Bar)<{ isTriggerSent: boolean }>`
+  height: 175px;
+  filter: ${({ isTriggerSent }) => (!isTriggerSent ? 'blur(4px)' : 'blur(0px)')};
+`;
+
+const Wrapper = styled.div`
+  background: rgba(30, 30, 38, 0.5);
+  padding: 0 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const StyledTitle = styled(Text)`
+  color: #dd2476; // ${colors.horizontal};
+  font-size: 11px;
+  margin: 12px 0 8px 0;
+`;
+
+const StyledText = styled(Text)`
+  margin: 0 15px 14px 15px;
+`;
+
+const MessageContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: absolute;
+  padding: 12px 15px 14px;
+  height: 64px;
+  background: ${colors.B20};
+  border-radius: 7px;
+`;
+
+function buildDisplayTitle(title) {
+  const dayMonth = title.split(',')[1].split('/');
+  const dateString = `1992-0${dayMonth[1]}-${dayMonth[0]}T00:00:00-00:00`;
+  const data = moment(dateString);
+
+  return `${data.format('dddd')}, ${data.format('MMMM')} ${data.format('Do')}`;
+}
+
+function getTitleStyle() {
+  let titleStyle = 'display: flex;';
+
+  titleStyle += 'justify-content: center;';
+  titleStyle += '; height: 17px';
+  titleStyle += '; margin-bottom: 4px';
+  titleStyle += '; border-width: 22px';
+  titleStyle += '; border-width: 22px';
+  titleStyle += `; color: ${colors.B60}`;
+
+  return titleStyle;
+}
+
+function getBodyStyle() {
+  let style = 'position: static';
+
+  style += '; display: flex;';
+  style += '; justify-content: center;';
+  style += '; height: 17px';
+  style += '; border-width: 22px';
+  style += `; color: #FF512F`;
+
+  return style;
+}
+
+function buildTitle(html: string, titleLines: any[]) {
+  let resHtml = html;
+
+  resHtml += '<div class="tooltip-title">';
+
+  titleLines.forEach(function (title) {
+    const displayTitle = buildDisplayTitle(title);
+
+    const titleStyle = getTitleStyle();
+
+    resHtml += `<span style="${titleStyle}" >${displayTitle}</th></tr>`;
```

Comment:
```suggestion
    resHtml += `<span style="${titleStyle}">${displayTitle}</span>`;
```
I think this should be a /span?

Reply:
Yea updated.